lang/java/reef-tang/tang/src/test/java/org/apache/reef/tang/TestClassLoaders.java

```diff
@@ -20,36 +20,40 @@
 
 import org.apache.reef.tang.exceptions.BindException;
 import org.apache.reef.tang.exceptions.InjectionException;
-import org.apache.reef.tang.exceptions.NameResolutionException;
 import org.apache.reef.tang.types.ClassNode;
 import org.junit.Assert;
+import org.junit.Test;
 
 import java.io.File;
+import java.io.FilenameFilter;
 import java.net.MalformedURLException;
 
 public class TestClassLoaders {
 
-  //  @Test
-  public void testOneJar() throws MalformedURLException,
-      ClassNotFoundException, NameResolutionException, BindException {
+  private static final FilenameFilter JAR_FILE_FILTER = new FilenameFilter() {
+    public boolean accept(final File dir, final String name) {
+      return name.endsWith(".jar");
```

Comment:
Thank you for your kind comments. You are right. But I'm a little bit reluctant to use hard-coded version. What about the following code? I think it's robust enough for your comments. 
```
-      return name.endsWith(".jar");
+      return name.matches("tang-test-jar(.*).jar");
```
To improve maintainability, I hope to use this manner instead.